test/DriverTest.php

```diff
@@ -349,4 +349,13 @@ private function getPermissionsFromStat(array $stat): string
     {
         return \substr(\decoct($stat["mode"]), 1);
     }
+
+    public function testChown(): \Generator
+    {
+        $fixtureDir = Fixture::path();
+
+        $original = "{$fixtureDir}/small.txt";
+        $user = fileowner($original);
```

Comment:
is there a non-blocking way to get the user id of a file owner?

Reply:
Not yet, I think. But doesn't matter for the test.